Source/RapyutaSimulationPlugins/Private/Tools/ROS2Spawnable.cpp

```diff
@@ -0,0 +1,42 @@
+//
+// Created by lng on 2021/10/13.
+//
+
+#include "Tools/ROS2Spawnable.h"
+
+// rclUE
+#include "Srvs/ROS2SpawnEntitySrv.h"
+
+void UROS2Spawnable::InitializeParameters(FROSSpawnEntity_Request Request)
+{
+    try{
+        SetName(Request.state_name);
+    } catch (const std::exception& e){
+        UE_LOG(LogTemp, Warning, TEXT("No state_name"));
+    }
+    try{
+        SetNamespace(Request.robot_namespace);
+    } catch (const std::exception& e){
+        UE_LOG(LogTemp, Warning, TEXT("No robot_namespace"));
+    }
+}
+
+void UROS2Spawnable::SetName(FString Name)
+{
+    ActorName = Name;
+}
+
+void UROS2Spawnable::SetNamespace(FString Namespace)
+{
+    ActorNamespace = Namespace;
+}
```

Comment:
The parameter could be of `const FString&`, and if we could check for its emptiness as you might want (as in above exceptions), we could log the warnings here-in I suppose.